applications/dashboard/models/class.usermodel.php

```diff
@@ -2791,17 +2813,27 @@ public function saveRoles($UserID, $RoleIDs, $RecordEvent) {
                 Logger::event(
                     'role_remove',
                     Logger::INFO,
-                    "{username} removed {toUsername} from the {role} role.",
-                    ['touserid' => $User->UserID, 'toUsername' => $User->Name, 'role' => $RoleName]
+                    "{toUsername} removed from the {role} role.",
+                    [
+                        'touserid' => $User->UserID,
```

Comment:
We added the `Logger::FIELD_TARGET_USERID` and `Logger::FIELD_TARGET_USERNAME` constants to normalize the user. FYI: It's `targetUserID` not `toUserID`.